docs/static/filebeat_modules/mysql/slowlog/pipeline.conf

```diff
@@ -0,0 +1,33 @@
+input {
```

Comment:
This config works for the ubuntu log, but returns a `_grokparsefailure` tag when I use https://github.com/elastic/beats/blob/master/filebeat/module/mysql/slowlog/test/mysql-darwin-brew-5.7.10.log:

`````
{
  "_index": "filebeat-2017.03.21",
  "_type": "log",
  "_id": "AVryZCvtu-hv5J-8aw-T",
  "_score": null,
  "_source": {
    "@timestamp": "2017-03-21T19:42:10.340Z",
    "offset": 210,
    "@version": "1",
    "input_type": "log",
    "beat": {
      "hostname": "Rhodas-MacBook-Pro.local",
      "name": "Rhodas-MacBook-Pro.local",
      "version": "5.3.0"
    },
    "host": "Rhodas-MacBook-Pro.local",
    "source": "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/filebeat-5.3.0-darwin-x86_64/log_examples/mysql_slowlog/mysql-darwin-brew-5.7.10.log",
    "message": "/usr/local/Cellar/mysql/5.7.10/bin/mysqld, Version: 5.7.10 (Homebrew). started with:\nTcp port: 3306  Unix socket: /tmp/mysql.sock\nTime                 Id Command    Argument\n# Time: 2016-12-12T11:54:16.601172Z",
    "type": "log",
    "tags": [
      "multiline",
      "beats_input_codec_multiline_applied",
      "_grokparsefailure"
    ]
  },
  "fields": {
    "@timestamp": [
      1490125330340
    ]
  },
  "sort": [
    1490125330340
  ]
}
`````